### PR TITLE
feat(route): Add Huggingface Daily Papers

### PR DESCRIPTION
**Pull request by zeyugao**
## 该 PR 相关 Issue / Involved Issue

None

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/huggingface/daily-papers
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [x] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

**Review by TonyRL — COMMENTED** [review]
Please use the namespace `huggingface` instead.
docs: https://docs.rsshub.app/joinus/script-standard.html

`programming.md` seems a tighter fit than `study.md`.

**Comment by zeyugao** [comment]
> Please use the namespace `huggingface` instead. docs: https://docs.rsshub.app/joinus/script-standard.html
> 
> `programming.md` seems a tighter fit than `study.md`.

Updated

File: docs/en/programming.md [hunk 1]
```diff
@@ -299,6 +299,12 @@ Subscribe to the updates (threads and submission) from a paritcular Hacker News
 
 <RouteEn author="hellodword" example="/hex-rays/news" path="/hex-rays/news" />
 
+## Huggingface
+
+### Daily Papers
+
+<Route author="zeyugao" example="/huggingface/daily-papers" path="/huggingface/daily-papers" />
```
Comment (TonyRL) [inline docs/en/programming.md:306]:
```suggestion
<RouteEn author="zeyugao" example="/huggingface/daily-papers" path="/huggingface/daily-papers" />
```

File: lib/v2/huggingface/daily-papers.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { body: response } = await got('https://huggingface.co/papers');
+    const $ = cheerio.load(response);
+    const papersJson = $('main > div[class="SVELTE_HYDRATER contents"]')[0].attribs['data-props'];
+    const papers = JSON.parse(papersJson);
```
Comment (TonyRL) [inline lib/v2/huggingface/daily-papers.js:9]: From my testing, the second match contains the required json
image

Maybe it's better to use to tighter match:

```suggestion
    const papers = $('main > div[data-target="DailyPapers"]').data('props');
```

Comment (zeyugao) [reply lib/v2/huggingface/daily-papers.js]: It seems that Huggingface adds a new banner just after my commit. So the automatic fetch by github-action works fine. I will update how we match the element.

**Pull request merged**